.semaphore/playwright-e2e.yml

```diff
@@ -77,6 +77,133 @@ blocks:
       jobs: *e2e-jobs
       epilogue: *e2e-epilogue
 
+  - name: "Windows x64: Playwright E2E Tests"
+    dependencies: []
+    task:
+      agent:
+        machine:
+          type: s1-prod-windows
+      prologue:
+        commands:
+          - $Env:PATH += ";C:\Program Files\Git\bin"
+          - $Env:PATH += ";C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\x64"
+          # Ensure npm and npx are available
+          - npm --version
+          - npx --version
+          - choco install gh -y
+          - $Env:PATH += ";C:\Program Files\GitHub CLI\"
+          - gh --version
+          - |
+            if (Test-Path "$HOME/.gh-token") {
+              $Env:GH_TOKEN = Get-Content "$HOME/.gh-token" -Raw | ForEach-Object { $_.Trim() }
+              if ([string]::IsNullOrEmpty($Env:GH_TOKEN)) {
+                  Write-Error "Token file exists but is empty"
+                  exit 1
+              }
+              gh config set prompt disabled
+              $Env:GH_TOKEN | gh auth login --with-token
+              gh auth status
+            } else {
+                Write-Error "Token file not found at $HOME/.gh-token"
+                exit 1
+            }
+          - |
+            if ($Env:SEMAPHORE_ORGANIZATION_URL -like "*.semaphoreci.com") {
+              Write-Output "Skipping Vault setup for Semaphore CI"
+            } else {
+              $Env:VAULT_ADDR = "https://vault.cireops.gcp.internal.confluent.cloud"
+              $token = $(vault write -field=token "auth/semaphore_self_hosted/login" role="default" jwt="$Env:SEMAPHORE_OIDC_TOKEN")
+              vault login -no-print token=$token
+            }
+          # Check if Docker is available for @local tests
+          - |
+            Write-Output "Checking Docker availability..."
+            try {
+              $dockerVersion = docker --version 2>$null
+              if ($LASTEXITCODE -eq 0) {
+                Write-Output "Docker is available: $dockerVersion"
+                make load-cached-docker-images
+              } else {
+                Write-Output "Docker is not available. @local tests will be skipped."
+              }
+            } catch {
+              Write-Output "Docker is not available. @local tests will be skipped."
```

Comment:
Hard-coding the `@local` skip for now, so this check doesn't actually affect the tests that run.